### DISCUSSION TITLE
Demo not working and nginxui.com documentation and other are down

### DISCUSSION THREAD

**Discussion by blender50** [0xJacky/nginx-ui#168] [General]
Hi folks,

I'm really looking forward to implementing this great looking tool on my Ubuntu 20.04 home-lab web server. Unfortunately at the time of this post the demo is intermittent and main site and documentation site(s) are throwing a 523 error from CloudFlare.

Just a head's up :-)

**Comment by 0xJacky** [comment]
Thank you for the heads-up! We've addressed the issue with our server provider, and now both the official website and the demo are back up and running smoothly.

**Reply by blender50** [reply]
**_Awesome!_**

**Reply by blender50** [reply]
Oops, it seems that I spoke too soon.....

This is what I'm seeing now...

<img width="874" alt="image" src="https://github.com/0xJacky/nginx-ui/assets/3241387/f6771bfb-c608-4d83-b4a0-4abdacf3f22b">

**Reply by 0xJacky** [reply]
Fixed

**Comment by vicpsl** [comment]
Hello,
the demo seems down again : 
![image](https://github.com/0xJacky/nginx-ui/assets/92852428/c8c55ff2-38b8-4543-b8c9-e94186f80c75)

Thank you

**Reply by 0xJacky** [reply]
Thanks, I fixed it just now.
